### PR TITLE
Snr distribution

### PR DESCRIPTION
**Pull request by JulienDoerner**
Here is the second try for a PR including a new free parameter in the SNR distribution. 
Additionaly changed some coding conventions in the pulsardistribution too.

### CODE REVIEW

**Comment by lukasmerten** [comment]
Looks much cleaner now. I assigned @rafaelab to review this as he started the review on the first version of this PR.

File: src/Source.cpp [hunk 1]
```diff
@@ -362,261 +362,275 @@ void SourceUniformCylinder::setDescription() {
...
 }
 
-double SourcePulsarDistribution::blur_theta(double theta_tilde, double r_tilde) const {
+double SourcePulsarDistribution::blurTheta(double thetaTilde, double rTilde) const {
 	Random &random = Random::instance();
-	double theta_corr = (random.rand()-0.5)*2*M_PI;
-	double tau = theta_corr*exp(-theta_blur*r_tilde);
-	return theta_tilde + tau;
+	double theta_corr = (random.rand() - 0.5) * 2 * M_PI;
+	double tau = theta_corr*exp(- thetaBlur * rTilde);
```
Comment (rafaelab) [inline src/Source.cpp:567]: Space between theta_corr * exp

File: src/Source.cpp [hunk 2]
```diff
@@ -362,261 +362,275 @@ void SourceUniformCylinder::setDescription() {
...
+	return thetaTilde + tau;
 }
 
-void SourcePulsarDistribution::set_frMax(double R, double b) {
-	frMax = pow(b, 2.) / (3*pow(R, 2.)*M_PI) * exp(-2.);
+void SourcePulsarDistribution::setFrMax(double R, double b) {
+	frMax = pow(b, 2.) / (3*pow(R, 2.) * M_PI) * exp(-2.);
```
Comment (rafaelab) [inline src/Source.cpp:572]: whitespace: 3 * pow

**Review by rafaelab — APPROVED** [review]
Hi @JulienDoerner 

Thanks! It looks good. There are just a couple of white spaces that could be fixed, as I indicated in my comments. I will already approve it, since there is nothing that requires attention at the moment.

**Comment by JulienDoerner** [comment]
@rafaelab, 
I have corrected the whitespaces.

**Pull request merged**